src/test/integration/ProjectsClient.test.ts

```diff
@@ -26,6 +26,28 @@ describe("ProjectsClient", () => {
     chai.expect(projectList).to.not.be.empty;
   });
 
+  it("should get project details from projectId", async () => {
+    const projectDetails: Project[] = await projectsAccessClient.getAll(accessToken,
```

Comment:
Since you are only interested in one project, you could verify projectDetails has a length greater than 0 then grab the first element instead of holding onto the list